pkg/process/checks/net.go

```diff
@@ -112,13 +123,35 @@ func (c *ConnectionsCheck) enrichConnections(conns []*model.Connection) []*model
 	return conns
 }
 
+func (c *ConnectionsCheck) diffTelemetry(tel *model.ConnectionsTelemetry) *model.CollectorConnectionsTelemetry {
+	if tel == nil {
```

Comment:
I think you need to set lastTelemetry here and also at the end of this function

Reply:
`lastTelemetry` is not a pointer.